hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -170,11 +177,17 @@ genCloseMutation (tx, _utxo) =
         newSigner <- verificationKeyHash <$> genVerificationKey
         pure $ ChangeRequiredSigners [newSigner]
     , SomeMutation MutateCloseUTxOHash . ChangeOutput 0 <$> mutateCloseUTxOHash
-    , SomeMutation MutateCloseContestationDeadline . ChangeOutput 0 <$> mutateClosedContestationDeadline
+    , SomeMutation MutateCloseContestationDeadline . ChangeOutput 0 <$> mutateClosedContestationDeadline Nothing
+    , SomeMutation MutateCloseContestationDeadline . ChangeOutput 0 <$> mutateClosedContestationDeadline (Just 0)
```

Comment:
Do not reuse the label. These are two different mutations / attack vectors, right?

Reply:
But do we have two different generators for this? Each `SomeMutation` is a generator and if both cases interest you (and are cases of `MutateCloseContestationDeadline`), and seemingly *any one of them* is enough.. then a simple generator is what you want here.

If you care about both being tasted -> two cases / two labels.